framework/wazuh/core/config/models/ssl_config.py

```diff
@@ -163,6 +160,33 @@ class APISSLConfig(WazuhConfigBaseModel):
     key: str
     cert: str
     use_ca: bool = False
-    ca: str = ""
+    ca: str = ''
     ssl_protocol: SSLProtocol = SSLProtocol.auto
-    ssl_ciphers: str = ""
+    ssl_ciphers: str = ''
+
+    @field_validator('ca')
```

Comment:
`key` and `cert` are not validated because they are being removed in https://github.com/wazuh/wazuh/issues/27776